src/activate/registerCommands.ts

```diff
@@ -310,3 +303,84 @@ export const openClineInNewTab = async ({ context, outputChannel }: Omit<Registe
 
 	return tabProvider
 }
+
+export const openSettingsInNewTab = async ({ context, outputChannel }: Omit<RegisterCommandOptions, "provider">) => {
+	const contextProxy = await ContextProxy.getInstance(context)
```

Comment:
For consistency with openClineInNewTab (which initializes CodeIndexManager), initialize it here as well to ensure any side effects or listeners are set up before resolving the webview.

```suggestion
const contextProxy = await ContextProxy.getInstance(context)
CodeIndexManager.getInstance(context)
```